src/quo2/foundations/colors_test.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.foundations.colors-test
+  (:require [cljs.test :refer-macros [deftest testing is]]
+            [quo2.foundations.colors :as colors]))
+
+(deftest test-color-resolver
```

Comment:
Added some tests as it is the core resolver of colors

Reply:
<img src="https://github.com/status-im/status-mobile/assets/46027/eca45040-7dac-4dfb-b391-477eb011a4d3" width="400" />